### DISCUSSION TITLE
Some troubles with different I/O

### DISCUSSION THREAD

**Discussion by UkrainianBanderasCat** [UkrainianBanderasCat/Digital-Logic-Sim2#44] [Q&A]
When I press the input pin (on the right side) it lights up red for a moment and then off again, but the inspector says that the pin is activated and the wire does not turn on at all and if you connect it to a 7 segment display the display will be permanently turned off until removed input on the right side, even if you connect the input on the left side

**Comment by firecerne** [comment]
Does it gives you an error message in the Log ?

**Reply by UkrainianBanderasCat** [reply]
No

**Reply by firecerne** [reply]
Is the question from a new functionnality you are currently working on ?

**Reply by UkrainianBanderasCat** [reply]
Yes

**Reply by firecerne** [reply]
If you send me your code, maybe I'll be able to help you

**Reply by UkrainianBanderasCat** [reply]
You can test yourself just in Prefabs -> Chip Editor -> Interaction(Hierarchy) -> Output Bar -> Chip Interface Editor(Script Inspector) -> Signal Prefab (Variable) change Output to Input and test by yourself.

**Reply by firecerne** [reply]
I'm not able to reproduce it, it doesn't do anything for me. Are you sure you didn't change anything else ?

**Reply by UkrainianBanderasCat** [reply]
Yes, When you click on Input on Output side it's just once blink a red light

**Reply by AlphaCodingPilot** [reply]
The same problem occurs when you connect a clock directly to an output pin.

**Reply by UkrainianBanderasCat** [reply]
Yey, I fixed that. And now it working but need to change chip packaging

**Reply by UkrainianBanderasCat** [reply]
Now we need to do a lot of work with chip saving

**Reply by UkrainianBanderasCat** [reply]
I will commit what I do
